src/cart/crt_iv.c

```diff
@@ -1684,8 +1680,8 @@ crt_iv_fetch(crt_iv_namespace_t ivns, uint32_t class_id,
 		if (put_needed)
 			iv_ops->ivo_on_put(ivns, iv_value, user_priv);
 		D_CDEBUG(rc == -DER_NOTLEADER, DB_ANY, DLOG_ERR,
```

Comment:
@jolivier23 is this correct?

Is DB_ANY a valid alternative to DLOG_ERR or should it be DLOG_INFO?